contrib/windows/msys_build.sh

```diff
@@ -203,7 +203,7 @@ fi
 echo 'FORCE_ASSERTIONS = 1' >> Make.user
 
 cat Make.user
-make -j3 VERBOSE=1 install
+make -j3 VERBOSE=1
```

Comment:
"general purpose" is overstating it, but it should be possible to run outside of appveyor. running `make install` isn't testing specific